test/integration/kubernetes_deploy_test.rb

```diff
@@ -632,6 +632,18 @@ def test_can_deploy_deployment_with_zero_replicas
     ])
   end
 
+  def test_can_deploy_deployment_with_partial_rollout_success
+    result = deploy_fixtures("slow-cloud", subset: ["configmap-data.yml", "web.yml.erb"])
```

Comment:
Since this test is the purpose of this fixture set, you shouldn't need to deploy a subset.